content/ja/docs/concepts/containers/runtime-class.md

```diff
@@ -152,4 +152,4 @@ PodのオーバーヘッドはRuntimeClass内の`overhead`フィールドによ
 - [RuntimeClassデザイン](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/585-runtime-class/README.md)
 - [RuntimeClassスケジューリングデザイン](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/585-runtime-class/README.md#runtimeclass-scheduling)
 - [Podオーバーヘッド](/docs/concepts/scheduling-eviction/pod-overhead/)のコンセプトを読む
-- [PodOverhead機能デザイン](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/20190226-pod-overhead.md)
+- [PodOverhead機能デザイン](https://github.com/kubernetes/enhancements/tree/master/keps/sig-node/688-pod-overhead)
```

Comment:
The link is same as the English page.
/lgtm

https://github.com/kubernetes/website/blob/d4c84372ab8c0a2af4fb4020cd85b2fd9fab6b5f/content/en/docs/concepts/containers/runtime-class.md?plain=1#L173